src/controllers/forgotpassword.js

```diff
@@ -0,0 +1,54 @@
+import jwt from 'jsonwebtoken'
+import { Employee } from '../../models'
+import nodemailer from 'nodemailer'
+
+// Setting a transporter
+const transporter = nodemailer.createTransport({
+  // host and port for MailHog
+  host: '0.0.0.0',
+  port: 1025
+})
+
+// forgotpassword controller function
+const forgotpassword = (req, res) => {
+  const { email } = req.body
+  // Responses for translations
+  const responseNoUserAvailable = req.t('forgot_password.failure')
+  const responseUserAvailable = req.t('forgot_password.success')
+  Employee.findOne({
+    where: {
+      email: email
+    }
+  }).then((user) => {
+    if (!user) {
+      res.status(400).json({
+        status: 400,
+        message: `${responseNoUserAvailable}`
+      })
+    } else {
+      // Generating Token for resetting password
+      const resetToken = jwt.sign({ email: email }, process.env.RESET_SECRET, { expiresIn: process.env.RESET_TOKEN_EXPIRES })
+      res.cookie('jwt', resetToken, { httpOnly: true, expiresIn: process.env.RESET_TOKEN_EXPIRES })
+
+      // Reset password url
+      const url = `http://localhost:${process.env.APP_PORT}/reset/${resetToken}`
+
```

Comment:
Please in production this will not work. I requested to change this.... the part of base URL should be an environment variable

Reply:
Well addressed. Please check again for approval!